service_discovery/dto/service_proposal.go

```diff
@@ -48,6 +50,14 @@ type ServiceProposal struct {
 	ProviderContacts []Contact `json:"provider_contacts"`
 }
 
+// SetProviderContact updates service proposal description with general data
+func (proposal *ServiceProposal) SetProviderContact(providerID identity.Identity, providerContact Contact) {
+	proposal.Format = "service-proposal/v1"
```

Comment:
Were we talking about moving it into the constant in some other PR? 
There are 8 occurrences of this string in the project.